debugging_protected_audience_api_services.md

```diff
@@ -55,57 +59,86 @@ misbehaving code and performance problems. This also means there is no need to
-collect logs, granular metrics and the plaintext request. These will be
-published to the AdTech's cloud store, where they can access this information.
-Publishing this will happen through [OpenTelemetry][7] and will use the same
-channels as mentioned in the [Monitoring Protected Audience API Services][6]
-document. Privacy impacting metrics that are usually exported with noise will be
-published without noise for debug consented requests.
+with. If the tokens match, the server will treat the request as a consented
+request. Otherwise the request will be treated as a normal request. The consent
+will be propagated to other Protected Audience API TEE servers in either case
+(whether the request is treated as consented or otherwise)
+
+When a server determines a request to be consented, it will collect logs,
+granular metrics and the plaintext request and response. These will be published
+to the AdTech's cloud store, where they can access this information. Publishing
+this will happen through [OpenTelemetry][7] and will use the same channels as
+mentioned in the [Monitoring Protected Audience API Services][6] document.
+Privacy impacting metrics that are usually exported with noise will be published
+without noise for consented requests.
 
 ### Reasons for using a secret debug-token
 
-- A secret debug-token helps filter out requests originating from other AdTechs
-  who may be debugging at the same time. This prevents intentional or
-  unintentional spam that would have made it difficult to isolate requests that
-  were sent by that particular AdTech.
+- A secret debug-token helps filter out requests originating from other
+    AdTechs who may be debugging at the same time. This prevents intentional or
+    unintentional spam that would have made it difficult to isolate requests
+    that were sent by that particular AdTech.
 - Without effective filtering, the associated cloud storage for an AdTech may
-  get filled with requests from other AdTechs, even though they didn't do any
-  debugging.
+    get filled with requests from other AdTechs, even though they didn't do any
+    debugging.
+
+### Provide Consent
+
+AdTech developers can give consent in Chrome and Android to make data from their
+client available for debugging. Care will be taken to prevent abuse of the
+setting. Note that this feature is targeted at the AdTech developers, not the
+general Chrome or Android user.
+
+- Chrome: AdTech can provide the consent at
+    `chrome://flags#protected-audience-debug-token` with a debug token.
+
+![Chrome UI](images/debugging_protected_audience_api_services_chrome_ui.png)
+
+- Android: The details are being designed and this explainer will be updated
+    in the future.
+
+### Configure TEE Servers
+
+To enable the AdTech consented debugging, every Protected Audience API TEE
+server should run with the following runtime flags:
+
+```
+ENABLE_OTEL_BASED_LOGGING = "true"
+CONSENTED_DEBUG_TOKEN = "[[secret debug token]]"
```

Comment:
May help to clarify what the config should be here. Eg. if debug token is "hello" in chrome, then does this need to be configured with "[[hello]]"?

Reply:
Changed to "secret debug token".